test/unit/full_text_search/searcher_test.rb

```diff
@@ -82,5 +82,41 @@ def test_syntax_error_query
       assert_equal([issue],
                    targets.collect(&:source_record).sort_by(&:id))
     end
+
+    def test_results_ordered_by_descending_registered_time
+      parameters = {
+        order_target: "registered_time",
+        order_type: "desc",
+        search_id: @project.id,
+        news: "1",
+        attachments: "0",
+        limit: "-1"
+      }
+      targets = search(parameters).records
+      searched_news_ids = targets.collect(&:source_id)
```

Comment:
How about compare records instead of only record IDs?
If this test is failed, we can only know record IDs. But we want to know `created_on` value for each record. If we use records instead of record IDs, failure message includes `created_on` values too.

```suggestion
      searched_news = targets.collect(&:source_record)
```

Reply:
fix: [a59d128](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/a59d12827884513ad2bf01fcd28407118cbd6ae4)
Thanks, I think I couldn't have this viewpoint when I wrote tests...